scipy/stats/tests/test_distributions.py

```diff
@@ -2171,12 +2171,17 @@ def test_infinite_input():
 def test_lomax_accuracy():
     # regression test for gh-4033
     p = stats.lomax.ppf(stats.lomax.cdf(1e-100,1),1)
-    assert_equal(p, 1e-100)
+    assert_almost_equal(p, 1e-100, decimal=15)
```

Comment:
This doesn't work, because ``assert_almost_equal`` does absolute comparison with 1e-15 precision here:

    assert_almost_equal(2e-100, 1e-100, decimal=15)

I suggest instead ``assert_allclose(p, 1e-100)``. That uses relative comparison with rtol=1e-7